test/helper.js

```diff
@@ -82,6 +82,11 @@ mocha.before(async () => {
   helper.intree = overwrites.intree = await helper.load('intree', overwrites);
   webServer = overwrites.server = await helper.load('server', overwrites);
 
+  helper.install = (id, name, repos) => overwrites.github.createInstall(id, name, repos);
```

Comment:
I would rather provide the tests with a way to access the fake github object, and have them call `createInstall` on that object.  I think `helper.install` is a little too vague - install what?